client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/operations/ReplaceIfUnmodifiedOperation.java

```diff
@@ -35,6 +36,7 @@ protected VersionedOperationResponse executeOperation(Transport transport) {
       transport.writeArray(key);
       transport.writeVInt(lifespan);
       transport.writeVInt(maxIdle);
+      codec.writeExpirationNanoTimes(transport, lifespanNanos, maxIdleNanos, internalFlags);
```

Comment:
Good point @danberindei about the > 30 days. I'll remove it if nobody has any objections.

Sending ms/ns by default will increase the message size. Isn't that important as well for HR?